js/search.js

```diff
@@ -846,6 +849,8 @@
 			else if (format === 'doublesou') tierSet = tierSet.slice(slices.DOU);
 			else if (format === 'doublesuu') tierSet = tierSet.slice(slices.DUU);
 			else if (format === 'doublesnu') tierSet = tierSet.slice(slices.DNU);
+			else if (isLetsGo && format.includes('ou')) tierSet = tierSet.slice(slices.OU);
```

Comment:
Why are you doing `includes`? Shouldn't you just strip off the preamble so you're actually just left with `ou`?

Reply:
```suggestion
```